notebooks/lesson10_clusters.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>should you tell them how many folds?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>